.coveralls.yml

```diff
@@ -1,2 +1,2 @@
 service_name: github
```

Comment:
not sure we want coveralls in figanos...
We won't have test for a whille because most of our outputs are figures...

Reply:
No worries. This can be left as is. It's not installed when building the package.

I can also comment out / remove the coveralls-related code blocks in the testing ensemble.